src/ica_api.c

```diff
@@ -3639,7 +3639,9 @@ unsigned int ica_aes_cmac_intermediate(const unsigned char *message,
 		       iv);
 
 	if (!rc)
-		stats_increment(ICA_STATS_AES_CMAC, ALGO_HW, ICA_DECRYPT);
+		stats_increment(ICA_STATS_AES_CMAC_128 +
+				aes_directed_fc_stats_ofs(function_code),
```

Comment:
Yes, I thought about that, but didn't want to use it because its not so readable/understandable ....
However, I am open to change it if thats what we want.